src/components/organisms/ClustersPane/ClustersPane.tsx

```diff
@@ -97,10 +114,9 @@ const ClustersPane = () => {
               type="primary"
               ghost
               loading={previewType === 'cluster' && previewLoader.isLoading}
```

Comment:
for some reason the loading animation is rendered over the label - not next to it!?

Reply:
Hmm, I saw as next to it. Let me check!

Reply:
I always see it next to the label. Can you please send me a ss?

Reply:
![image](https://user-images.githubusercontent.com/1917063/131319073-211329f8-8282-4088-a56e-18c050c1413a.png)
you can see the spinning circle over the initial 'L' - this is only temporary - after a second or two the label moves to the right and the button looks ok...

Reply:
I can confirm this, it happens because the UI freezes while the preview is loading. Probably that's why the animation on Skeletons doesn't work either..

Reply:
ok thanks @catalinhoha - then let's live with it for now and consider moving the loading of cluster resources to the main thread in a separate issue!